### PR TITLE
Update defender-advanced-threat-protection.md

### PR DESCRIPTION
**Pull request by kasiak-msft**
Updated some branding. Removed reference to "formerly known as Windows Defender for Endpoint" as this was never the name of the product.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@mestew @aczechowski Would one of you be able to review the proposed update? Thanks.

**Comment by PRMerger7** [comment]
@mestew : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mestew — APPROVED** [review]

**Comment by mestew** [comment]
#sign-off

File: memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ ms.localizationpriority: medium
...
 
-Microsoft Defender for Endpoint is a service in the [Microsoft Defender Security Center](https://securitycenter.windows.com). By adding and deploying a client onboarding configuration file, Configuration Manager can monitor deployment status and Microsoft Defender for Endpoint agent health. Microsoft Defender for Endpoint is supported on PCs running the Configuration Manager client or [managed by Microsoft Intune](/intune/protect/advanced-threat-protection).
+Microsoft Defender for Endpoint's cloud based portal is [Microsoft Defender Security Center](https://securitycenter.windows.com). By adding and deploying a client onboarding configuration file, Configuration Manager can monitor deployment status and Microsoft Defender for Endpoint agent health. Microsoft Defender for Endpoint is supported on PCs running the Configuration Manager client or [managed by Microsoft Intune](/intune/protect/advanced-threat-protection).
```
Comment (ShannonLeavitt) [inline memdocs/configmgr/protect/deploy-use/defender-advanced-threat-protection.md:20]:
```suggestion
Microsoft Defender for Endpoint's cloud-based portal is [Microsoft Defender Security Center](https://securitycenter.windows.com). By adding and deploying a client onboarding configuration file, Configuration Manager can monitor deployment status and Microsoft Defender for Endpoint agent health. Microsoft Defender for Endpoint is supported on PCs running the Configuration Manager client or [managed by Microsoft Intune](/intune/protect/advanced-threat-protection).
```
Fixed hyphenation

**Comment by PRMerger19** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**